docker/mint.properties

```diff
@@ -0,0 +1 @@
+mintr_url=http://mintr_mintr_1:8888
```

Comment:
Annoyingly, this is changing in recent docker compose and we need a solution to deal with this at some point. If these are on the same network can you get away with `http://mintr:8888` - I think that should work and be more robust

Reply:
Ah yeah we're actually already doing that for the app url anyway.